apps/stats/src/views/Stats/Web.tsx

```
@@ -0,0 +1,98 @@
+                <Select value={`${range}`} onValueChange={(value) => {
+                    setRange(Number(value));
+                }}>
+                    <SelectTrigger className="w-[180px]">
+                        <SelectValue placeholder="Select a period" />
+                    </SelectTrigger>
+                    <SelectContent>
+                        <SelectGroup>
+                            <SelectLabel>Period</SelectLabel>
+                            {RANGE_OPTIONS.map((option) => {
+                                return (
+                                    <SelectItem value={`${option.value}`}>{option.name}</SelectItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)